### PR TITLE
Create dummy Login component

### PR DESCRIPTION
**Pull request by SeanMcP**
Change default /admin route to login

### CODE REVIEW

File: app/javascript/packs/src/components/AdminLogin.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React, {Component} from 'react';
+
+export default class AdminLogin extends Component{
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      password: ''
+    }
+    this.handleName = this.handleName.bind(this);
```
Comment (IsaacHardy) [inline app/javascript/packs/src/components/AdminLogin.js]: Not need if using arrow functions.

File: app/javascript/packs/src/components/AdminLogin.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    this.handlePassword = this.handlePassword.bind(this);
```
Comment (IsaacHardy) [inline app/javascript/packs/src/components/AdminLogin.js]: Not need if using arrow functions.

File: app/javascript/packs/src/components/AdminLogin.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    this.handleSubmit = this.handleSubmit.bind(this);
```
Comment (IsaacHardy) [inline app/javascript/packs/src/components/AdminLogin.js]: Not need if using arrow functions.

File: app/javascript/packs/src/components/AdminLogin.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  }
+  handleName = e => {
+    this.setState({ name: e.target.value });
+  }
+  handlePassword = e => {
+    this.setState({ password: e.target.value });
+  }
+  handleSubmit = () => {
+    console.log('this.state:', this.state);
```
Comment (IsaacHardy) [inline app/javascript/packs/src/components/AdminLogin.js]: Remove logs from Pull requests.

**Comment by Rdore88** [comment]
⛵️

**Pull request merged**